lib/Test/Builder.pm

```diff
@@ -55,6 +58,8 @@ sub _add_ts_hooks {
     # in rare cases.
     my $epkgr = \$self->{Exported_To};
 
+    #$hub->add_context_aquire(sub {$_[0]->{level} += $Level - 1});
```

Comment:
oops, need to get rid of this. Initially I thought a hub-specific callback was in order, but that is wrong since it means only some tests would be effected, but $Level is global, so it should have a global effect.